src/main/java/seedu/club/model/task/Status.java

```diff
@@ -0,0 +1,36 @@
+package seedu.club.model.task;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents the status of a Task
+ */
+public class Status {
+
```

Comment:
Possibility to use enums instead in future releases to reduce complexity

Reply:
Good idea! Will do that in future releases